### ISSUE TITLE
Missing white-space between issue icon and label in global comment (GitLab)

### ISSUE THREAD

**Issue by p3k** [mc1arke/sonarqube-community-branch-plugin#421]
**Describe the enhancement**

The current output of the global comment in GitLab is missing some white-space between the number of bugs, vulnerabilities etc. and the label:

![image](https://user-images.githubusercontent.com/16631/130033458-f2881a1a-e0ff-4f29-a20b-761a67a900a1.png)

Here is the corresponding HTML code from dev tools:

![image](https://user-images.githubusercontent.com/16631/130033959-4931929f-f9b9-46f9-aba4-a1780ed746c4.png)

The red arrow points to actual white-space in the DOM but this is obviously not rendered.

In the [plugin code](https://github.com/mc1arke/sonarqube-community-branch-plugin/blob/1.8.0/src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/AnalysisDetails.java#L186) I found additions of `new Text(" ")` instances at the correct positions, but it seems these are not rendered either…(?)

**Expected behavior**

What could be done to add the missing white-space to achieve something like this – IMHO better looking _and_ readable – display:

![image](https://user-images.githubusercontent.com/16631/130035109-ef3c0aa2-f5b4-4cb0-a2b7-730987a0d395.png)

**Software Versions**
- GitLab Version: 14.1.2 (community / self-hosted)
- SonarQube Version: 8.9.2.46101 (community / self-hosted)
- Plugin Version: 1.8.0
- Browser: Chromium 91.0.4472.101 and Firefox 91.0

**Comment by kianms10310** [comment]
I had the smae issue, so I modified that.

```java
@Override
    public Formatter<Text> textFormatter() {
        return new BaseFormatter<Text>() {
            @Override
            public String format(Text node, FormatterFactory formatterFactory) {
                if (StringUtils.isBlank(node.getContent())) {
                    return " ";
                }
                return htmlEscaper().escape(node.getContent()).trim();
            }
        };

```
new TEXT(" ") is used, whitespace is always trim()

**Comment by mc1arke** [comment]
Released in 1.11.0
